test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -200,6 +200,8 @@ protected void afterIfFailed(List<Throwable> errors) {
                 Files.walk(path).forEach(file -> {
                     if (file.toString().endsWith(".yml")) {
                         addSuite(root, file, files);
+                    } else if (file.toString().endsWith(".yaml")) {
+                        throw new IllegalArgumentException("yaml files are no longer supported: " + file);
```

Comment:
I started with that but we have a bunch of txt files in there now. I could remove the txt files or fail on any non-yml, non-txt files. That'd be a stronger assertion, but I wanted to get some kind of assertion in now because I'd already made the mistake.